packages/commonwealth/server/models/thread.ts

```diff
@@ -168,6 +168,10 @@ export default (
       foreignKey: 'thread_id',
       as: 'reactions',
     });
+    models.Thread.hasMany(models.Comment, {
```

Comment:
Do we still need to add this?

Reply:
Not necessary, we can remove. This however won't do anything and there is still a relation b/w comments and threads from the db POV. Lmk if i should remove!

Reply:
Please remove them until we implement a formal association (right now it's only an implicit contract rather than a proper relation at the schema level).

Reply:
@jnaviask was just checking on schema for. different clean up ticker

We do have:
```
    "Comments_thread_id_fkey" FOREIGN KEY (thread_id) REFERENCES "Threads"(id)
```

<img width="1239" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/4791635/45926137-8290-4711-bbde-c46147593779">

Reply:
Came from #3996.